packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -587,6 +617,9 @@ class Tooltip extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(PolymerE
   /** @private */
   __tooltipRenderer(root) {
     root.textContent = typeof this.generator === 'function' ? this.generator(this.context) : this.text;
+
+    // Update the sr-only label text content
+    this._overlayContent = root.textContent;
```

Comment:
Instead of duplicating the content, would it be an option to remove the shadow root from `<vaadin-tooltip>`, so that the overlay itself would be inside the tooltip's light dom when closed?

Reply:
In theory this can be done, but how would we announce the tooltip after the overlay opens?

Reply:
True, better this way